koji-scm-lib/src/main/java/org/fakekoji/xmlrpc/server/JavaServerConstants.java

```diff
@@ -0,0 +1,13 @@
+package org.fakekoji.xmlrpc.server;
+
+public class JavaServerConstants {
+    public static final String xPortAxiom = "XPORT";
+    public static final String dPortAxiom = "DPORT";
+    public static final int DFAULT_RP2C_PORT = 9848;
+    public static final int DFAULT_DWNLD_PORT = deductDwPort(DFAULT_RP2C_PORT);
+
+    public static int deductDwPort(int xport) {
```

Comment:
do we really need method for this? and public?

This is imho sufficient and more readable
```
public static final int DFAULT_RP2C_PORT = 9848;
public static final int DFAULT_DWNLD_PORT = DFAULT_RP2C_PORT + 1;
```